cartridges/cloudinary_sg_changes/cartridge/templates/default/slots/category/catlanding-banner.isml

```diff
@@ -24,12 +24,14 @@
             <a href="${URLUtils.http('Search-Show', 'cgid', category.ID)}" title="${category.displayName}">
                 <isif condition="${!empty(catBGImageUrl)}">
                     <img
-                        class="cld-responsive"
-                        data-src="${catBGImageUrl}"
-                        <isif condition="${typeof(catBGImageUrl.isResponsive) != 'undefined' && !empty(catBGImageUrl.isResponsive) && !catBGImageUrl.isResponsive}">
-                            <isif condition="${catBGImageUrl.srcset}">
-                                srcset="${catBGImageUrl.srcset}"
+                        class="${cldBannerImage.isResponsive ? 'cld-responsive' : ''}"
+                        <isif condition="${!cldBannerImage.isResponsive}">
+                            src="${catBGImageUrl}"
+                            <isif condition="${cldBannerImage.srcset}">
```

Comment:
we should have the 'sizes' attribute as well

Reply:
It's not there, from the start. If you really need we can add that.

Reply:
yes, it's mandatory